tests/framework/extensions/clusters/import.go

```diff
@@ -137,7 +140,10 @@ func ImportCluster(client *rancher.Client, cluster *management.Cluster, rest *re
 						{
 							Name:    "kubectl",
 							Image:   "rancher/shell:v0.1.10",
-							Command: []string{"kubectl", "apply", "--filename=" + token.ManifestURL},
+							Command: []string{"/bin/sh", "-c"},
+							Args: []string{
+								fmt.Sprintf("count=0\n until\n [ $status == 0 ] || [$count -eq 4] \n do\n apk --no-cache add curl\n status=$?\n count=$((count+1))\n done;  %s;", token.InsecureCommand),
```

Comment:
wget is included in the shell image, it would be faster to use wget.

Reply:
Updated